tasks/wikimedium.10M.nostopwords.tasks

```diff
@@ -14409,8 +14409,6 @@ BrowseDayOfYearTaxoFacets: *:* +facets:DayOfYear.taxonomy
 BrowseDateSSDVFacets: *:* +facets:Date.sortedset
 BrowseMonthSSDVFacets: *:* +facets:Month.sortedset
 BrowseDayOfYearSSDVFacets: *:* +facets:DayOfYear.sortedset
-BrowseRandomLabelTaxoFacets: *:* +facets:RandomLabel.taxonomy
```

Comment:
Need to remove these two tasks as index name was getting too long when I tried to include sorting

Reply:
I thought @zhaih had made a change to stop building such massive filenames -- maybe it's opt-in (you have to provide your own index name) or so?

Reply:
Thanks @mikemccand for the suggestion! I'm not aware of this feature and it does seems like an opt-in to me, but I'll dig around and report back.

Reply:
That's actually not added by me, it's there for long time: https://github.com/mikemccand/luceneutil/blob/master/src/python/competition.py#L155 

I think just set the name will avoid the overlong auto-generated name.